lib/pf/cmd/pf/pfmon.pm

```diff
@@ -0,0 +1,99 @@
+package pf::cmd::pf::pfmon;
+
+=head1 NAME
+
+pf::cmd::pf::pfmon -
+
+=head1 SYNOPSIS
+
+pfcmd pfmon <task> [options...]
+
+=head1 DESCRIPTION
+
+pf::cmd::pf::pfmon
+
+=cut
+
+use strict;
+use warnings;
+use pf::config::pfmon qw(%ConfigPfmon);
+use pf::constants::exit_code qw($EXIT_SUCCESS);
+use pf::constants;
+use pf::factory::pfmon::task;
+use base qw(pf::cmd);
+
+sub parseArgs {
+    my ($self) = @_;
+    my ($task_id, @args) = $self->args;
+    return 0 unless defined $task_id;
+    unless (exists $ConfigPfmon{$task_id}) {
+        print STDERR "$task_id is not a valid task\n";
+        return 0;
+    }
+    unless ($self->_parse_attributes(@args)) {
+        return 0;
+    }
+    $self->{task_id}  = $task_id;
+    return 1;
+}
+
+sub _run {
+    my ($self) = @_;
+    my $task_id = $self->{task_id};
+    my $params = $self->{params};
+    my $task = pf::factory::pfmon::task->new($task_id, $params);
+    $task->run();
+    return $EXIT_SUCCESS;
+}
+
+=head2 _parse_attributes
+
+parse and validate the arguments for 'pfcmd pfmon <task> [args]' command
+
+=cut
+
+sub _parse_attributes {
+    my ($self,@attributes) = @_;
+    my %params;
+    for my $attribute (@attributes) {
+        if($attribute =~ /^([a-zA-Z0-9_-]+)=(.*)$/ ) {
+            $params{$1} = $2;
+        } else {
+            print STDERR "$attribute is badily formatted\n";
```

Comment:
badily sounds like a mix between bad and baladi

I can propose baldily, that would be a mix between bald and baladi
*insert video of zammit dancing here*